src/table/cells/handleKeyPress.js

```diff
@@ -0,0 +1,74 @@
+const moveFocus = (tableRows, nextRow, nextCol) => {
+  const nextCell = tableRows[nextRow].getElementsByClassName('sn-table-cell')[nextCol];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const arrowKeysNavigation = (evt, rowAndColumn, rowIndex, colIndex) => {
+  let nextRow = rowIndex;
+  let nextCol = colIndex;
+
+  switch (evt.key) {
+    case 'ArrowDown':
+      if (rowIndex + 1 < rowAndColumn.tableRowSize) nextRow = ++nextRow;
+      break;
+    case 'ArrowUp':
+      if (rowIndex > 0) nextRow = --nextRow;
+      break;
+    case 'ArrowRight':
+      if (colIndex < rowAndColumn.tableColumnSize - 1) nextCol = ++nextCol;
+      break;
+    case 'ArrowLeft':
+      if (colIndex > 0) nextCol = --nextCol;
+      break;
+    default:
+  }
+
+  return {
+    tableRows: rowAndColumn.tableRows,
+    nextRow,
+    nextCol,
+  };
+};
+
+const tableRowAndColumn = (rootElement) => {
+  const tableRows = rootElement.getElementsByClassName('sn-table-row');
+  const tableRowSize = tableRows.length;
+
+  const headCells = rootElement.getElementsByClassName('sn-table-head-cell');
+  const tableColumnSize = headCells.length;
+
+  return { tableRows, tableRowSize, tableColumnSize };
```

Comment:
And no need to call these `table...` since we know it is in the table.
```suggestion
  return { rows, rowCount, rowWidth };
```

Reply:
yeah, sure